src/policy/policy.cpp

```diff
@@ -78,7 +78,7 @@ bool IsStandardTx(const CTransaction& tx, std::string& reason)
         // bytes of scriptSig, which we round off to 1650 bytes for some minor
         // future-proofing. That's also enough to spend a 20-of-20
         // CHECKMULTISIG scriptPubKey, though such a scriptPubKey is not
-        // considered standard)
```

Comment:
When touching this comment, can you please also move the '(520 byte limit)' above before the dot?